.spi.yml

```diff
@@ -0,0 +1,19 @@
+# This is manifest file for the Swift Package Index for it to 
+# auto-generate and host DocC documentation.
+# For reference see https://swiftpackageindex.com/swiftpackageindex/spimanifest/documentation/spimanifest/commonusecases#Host-DocC-documentation-in-the-Swift-Package-Index.
+
+version: 1
+builder:
+  configs:
+    - documentation_targets: 
+      - SwiftSyntax    
```

Comment:
Sorry, I could have mentioned this earlier, but could you add a comment that the first target in this list is the primary target. I have a tendency to order things alphabetically and will forget about it 🙈 

Also, could you squash your commits before merging?

Reply:
Added comment and squashed

Reply:
Thank you ❤️